.php_cs.dist

```diff
@@ -132,9 +132,9 @@ return Config::create()
             'align' => 'left',
         ],
         'php_unit_dedicate_assert' => true,
-        'php_unit_method_casing' => [
+        /*'php_unit_method_casing' => [
             'case' => 'snake_case',
-        ],
+        ],*/
```

Comment:
Disabled this because it was doing something like:
```diff
-    public function test_it_can_parse_PHP_doc_blocks(string $docBlock, string $expected): void
+    public function test_it_can_parse__ph_p_doc_blocks(string $docBlock, string $expected): void
```

Reply:
indeed should be `test_it_can_parse__php_doc_blocks`: we can easier disable it or remove the cache and enable it again to fix all those cases. I'm fine either way